CHANGES.rst

```diff
@@ -1,3 +1,13 @@
+1.3.2 (February 3rd, 2021)
+==========================
+Fixes a bug in cases where the number of targets is larger than
+the number of samples in each target.
+
+1.3 (December 2, 2020)
+=========================
+This version includes changes to the documentation, to refer to the
+published paper.
+
```

Comment:
These changes are unrelated to the main purpose of this PR, but may as well merge them here as well.